### ISSUE TITLE
Cannot read properties of null (reading 'setLoadingTab')

### ISSUE THREAD

**Issue by alessandrojbr** [VemtoOrg/vemto2-issues#106]
**Error Message:**
Cannot read properties of null (reading 'setLoadingTab')

**Stack:**
```
TypeError: Cannot read properties of null (reading 'setLoadingTab')
    at h (file:///C:/Users/alessandro.ribeiro/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.3863d8c7.js:1261:30127)
    at file:///C:/Users/alessandro.ribeiro/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.3863d8c7.js:1261:30054
    at dr (file:///C:/Users/alessandro.ribeiro/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.3863d8c7.js:93:12282)
    at ma (file:///C:/Users/alessandro.ribeiro/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.3863d8c7.js:93:12605)
    at ta (file:///C:/Users/alessandro.ribeiro/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.3863d8c7.js:93:12547)
```

---

Vemto 2.0.8
win32 10.0.22631
Locale: pt-BR
App version: 2.0.8

**Comment by TiagoSilvaPereira** [comment]
@alessandrojbr thanks for reporting

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 2.0.10
